### ISSUE TITLE
The syntax checking for .gitignore is incorrectly reporting errors.

### ISSUE THREAD

**Issue by zxcvbnm3057** [foxundermoon/vs-shell-format#379]
It reported an error with message `a command can only contain words and redirects; encountered (`

The gitignore was from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore. And the exact error position is 276:14

```
*- [Bb]ackup ([0-9]).rdl
*- [Bb]ackup ([0-9][0-9]).rdl
```

please fix this

**Comment by kbrnsr-nmbu** [comment]
Yeah, this project seems to be abandoned at this point with the PRs stuck in limbo and last release in 2023
